7.0.0/postgresql/org_data_fetch.sh

```
@@ -0,0 +1,76 @@
+
+# Execute the query and capture output in a raw format
+psql -h "$DB_HOST" -p "$DB_PORT" -d "$DB_NAME" -U "$DB_USER" -c "$FETCH_QUERY" -t -A -F $'\t' 2>error.log > "$RAW_OUTPUT"
+
+# Check for errors
+if [[ $? -ne 0 ]]; then
+  echo "Failed to fetch data. PostgreSQL error log:"
+  cat error.log
+  exit 1
+fi
+
+> "$EXPORT_FILE" # Clear previous data.
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)